src/scripts/main.js

```diff
@@ -1 +1,60 @@
 'use strict';
+
+const html = document.querySelector('html');
+const body = document.querySelector('body');
+
+function createMessage(type, message) {
+  body.insertAdjacentHTML('beforeend', `
+      <div class="${type}" data-qa="notification">
+        ${message}
+      </div>
+      `);
+};
+
+const promise1 = new Promise((resolve, reject) => {
+  html.addEventListener('click', () => {
+    resolve();
+  });
+
+  setTimeout(() => {
+    reject(new Error());
+  }, 3000);
+});
+
+const promise2 = new Promise(resolve => {
+  html.addEventListener('click', () => {
+    resolve();
+  });
+
+  html.addEventListener('contextmenu', (e) => {
+    e.preventDefault();
+    resolve();
+  });
```

Comment:
Can be simplify with one listener and [`event.button`](https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/button) checker